### DISCUSSION TITLE
New aliases feature doesn't work with SVG import

### DISCUSSION THREAD

**Discussion by in-in** [vercel/next.js#12866] [Help]
Does the new aliases feature allow me to use aliases for SVG import?
```json
// jsconfig.json
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "~/*": ["src/*"],
      "@assets/*": ["src/assets/*"],
      "@helpers/*": ["src/helpers/*"]
    }
  }
}
```
```js
import Cart from '@assets/icons/cart.svg';

<Cart/>
```

file structure
```shell
.
├── pages
└── src
    ├── assets
    │   ├── fonts
    │   ├── icons
    │   │   ├── cart.svg
    │   └── images
    ├── components
    ├── helpers
```

And I get the following error:
```
Cannot find module '@assets/icons/cart.svg'
```

Aliases work fine with `js`, `jsx` files. 
And SVG import work without aliases 
```
import Cart from '../../assets/icons/cart.svg'
```

"next": "9.4.0"

**Comment by kkfuri** [comment]
Having this same problem with next 9.4.0.

**Reply by in-in** [reply]
I've opened the issue, and if you're still interested in this problem, please add :+1:  to that issue.
https://github.com/vercel/next.js/issues/13584

**Comment by manueltruepill** [comment]
Based on this [answer](https://github.com/vercel/next.js/issues/13584#issuecomment-636370600) I switched from `babel-plugin-inline-react-svg` to `@svgr/webpack` and path aliases for SVG files is working now.

See this [guide](https://dev.to/dolearning/importing-svgs-to-next-js-nna)
